File: chatwoot/typescript-node/src/DeleteAnAccountAgentBotExample.ts
import * as fs from 'fs';
import api from "chatwoot_client"
import models from "chatwoot_client"

const apiCaller = new api.AccountAgentBotsApi();
apiCaller.setApiKey(api.AccountAgentBotsApiApiKeys.userApiKey, "USER_API_KEY");
// apiCaller.setApiKey(api.AccountAgentBotsApiApiKeys.agentBotApiKey, "AGENT_BOT_API_KEY");
// apiCaller.setApiKey(api.AccountAgentBotsApiApiKeys.platformAppApiKey, "PLATFORM_APP_API_KEY");

apiCaller.deleteAnAccountAgentBot(
  undefined, // accountId
  undefined, // id
).catch(error => {
  console.log("Exception when calling AccountAgentBotsApi#deleteAnAccountAgentBot:");
  console.log(error.body);
});

File: chatwoot/kotlin/src/main/kotlin/oseg/chatwoot_examples/DeleteAnAccountAgentBotExample.kt
package oseg.chatwoot_examples

import com.chatwoot.client.infrastructure.*
import com.chatwoot.client.apis.*
import com.chatwoot.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class DeleteAnAccountAgentBotExample
{
    fun deleteAnAccountAgentBot()
    {
        ApiClient.apiKey["api_access_token"] = "USER_API_KEY"
        // ApiClient.apiKey["api_access_token"] = "AGENT_BOT_API_KEY"
        // ApiClient.apiKey["api_access_token"] = "PLATFORM_APP_API_KEY"

        try
        {
            AccountAgentBotsApi().deleteAnAccountAgentBot(
                accountId = 0,
                id = 0,
            )
        } catch (e: ClientException) {
            println("4xx response calling AccountAgentBotsApi#deleteAnAccountAgentBot")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling AccountAgentBotsApi#deleteAnAccountAgentBot")
            e.printStackTrace()
        }
    }
}
